apps/studio/next.config.js

```diff
@@ -530,10 +524,6 @@ const nextConfig = {
         source: '/favicon/:slug*',
         headers: [{ key: 'cache-control', value: 'public, max-age=86400' }],
       },
-      {
-        source: '/(.*).ts',
-        headers: [{ key: 'content-type', value: 'text/typescript' }],
-      },
```

Comment:
You might still want this, Vercel defaults `*.ts` file content type to `video/mp2t`. Shouldn't affect functionality, but it will send with that content type.